src/components/Header/Header.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import useWindowWidth from "../../../hooks/useWindowResize.js";
+import HeaderLogo from "./HeaderLogo/HeaderLogo.jsx";
+import HeaderSearchBar from "./HeaderSearchBar/HeaderSerchBar.jsx";
+import HeaderCart from "./HeaderCart/HeaderCart.jsx";
+import './Header.css';
+
+export default function Header(props) {
+    const {
+        headerCartProducts,
+        setHeaderCartProducts,
+        isOpenHeaderCartModal,
+        setIsOpenHeaderCartModal,
+        isCartShakingAnimation,
+        setIsCartShakingAnimation,
+        onChangeHeaderInputValue,
+        headerProductSearchingInput,
+        setHeaderProductSearchingInput,
+    } = props;
+
+    const [isOpenSearchBar, setIsOpenSearchBar] = useState(true);
+    const width = useWindowWidth();
+    const isMobile = width <= 480;
+
+    useEffect(() => {
+        if (isMobile) {
+            setIsOpenSearchBar(false);
+        }
+
+    }, [width]);
+
+    function showInputOnClick() {
```

Comment:
Знову назви. Який інпут?

```suggestion
    function handleIsOpenSearchBar() {
```